recipes/mender-orchestrator-support/debian-master/control

```diff
@@ -0,0 +1,26 @@
+Source: mender-orchestrator-support
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-orchestrator-support
+Architecture: all
+Depends: dbus, curl, mender-orchestrator-core, mender-update, mender-auth, ${misc:Depends}
```

Comment:
We should require the same version of `mender-orchestrator-core` as the version of `mender-orchestrator-support`, shouldn't we?

Reply:
I see. :+1:  Then I guess time will tell what the version requirements here will be.